File: problem_solving_for_beginners/bitwise/c-programming/toggle_nth_bit.c
#include<stdio.h>
/***
 * main - Toggle nth bit of a given number using bitwise
 */
int main(void)
{
	int num, n, res;

    printf("Enter number and nth bit you want to disable: ");
	scanf("%d%d", &num,&n);

	//Write your code here
    res = num ^ (1 << (n - 1));

    printf("%d\n", res);

	return 0;
}

File: problem_solving_for_beginners/bitwise/python/toggle_nth_bit.py
#!/usr/bin/python3
"""
    Toggle nth bit of a given number using
    bitwise XOR and LEFT SHIFT operators
"""
num = int(input("Enter a number: "))
n = int(input("Enter nth bit to toggle: "))

n = n - 1
res =  num ^ (1 << n)

print(res)